conf/janus.plugin.streaming.jcfg.sample.in

```diff
@@ -249,4 +249,31 @@ file-ondemand-sample: {
 	#rtsp_user = "username"
 	#rtsp_pwd = "password"
 	#secret = "adminpwd"
+
+	# After n seconds passed and no media assumed, the RTSP server has
+	# gone and schedule a reconnect.
+	#
+	# Default value is 5 seconds.
+	#reconnect_delay = 5
```

Comment:
This should have the `rtsp_` prefix, as it's only valid for RTSP mountpoints. People may think a property called `reconnect_delay` applies to any kind of mountpoint instead, while it doesn't.